GoRogue/MapGeneration/Region.Properties.cs

```diff
@@ -28,47 +28,47 @@ public partial class Region
         /// <summary>
         /// All of the boundary points of this region
         /// </summary>
-        public IReadOnlyList<Point> OuterPoints => _outerPoints.AsReadOnly();
-        private List<Point> _outerPoints = new List<Point>();
+        public Area OuterPoints => _outerPoints;
```

Comment:
Ideally, we should expose this as `IReadOnlyArea`:
```suggestion
        public IReadOnlyArea OuterPoints => _outerPoints;
```

The current interface can allow a user outside the class to directly modify this list, which since this was previously exposed as `IReadOnlyList` I'm asssuming is undesireable?

Reply:
D'oh! You're right. Fixed.